suffuse/node_fuse.go

```diff
@@ -3,22 +3,92 @@ package suffuse
 import (
   "github.com/suffuse/go-suffuse/suffuse/xattr"
   "os"
+  "time"
   "golang.org/x/net/context"
   "bazil.org/fuse/fs"
   f "bazil.org/fuse"
 )
 
+// Access checks whether the calling context has permission for
```

Comment:
In my editor it's very easy to add/remove // comments, much less so for /* */. Also I assume /* */ doesn't nest in go like it doesn't anywhere except scala. I find // to be an overall better commenting experience.